### ISSUE TITLE
Any collapsable tree view widget?

### ISSUE THREAD

**Issue by deeplook** [jupyter-widgets/ipywidgets#2345]
I'm looking for a tree view widget like this, but as an ipywidget without the need for runninng JupyterLab: https://github.com/gnestor/jupyterlab_json. Does anything like this exist or is it on the roadmap?

**Comment by martinRenou** [comment]
I worked on a small tree widget: https://github.com/QuantStack/ipytree and there are some examples notebooks https://github.com/QuantStack/ipytree/tree/master/examples. I plan to package it for conda at some point. And I would like to add context menus and other fancy features.

**Comment by martinRenou** [comment]
You can try it on binder before installing it: https://mybinder.org/v2/gh/QuantStack/ipytree/master :)

**Comment by deeplook** [comment]
I just tried very briefly. Will have to do more, but ipyleaflet seems to be missing when running in binder...

**Comment by martinRenou** [comment]
Yeah sorry the repo needs an `environment.yml` file specifying what the binder needs, but you can try the second notebook `ipytree.ipynb` which should work just fine :)

**Comment by deeplook** [comment]
I know, I'm doing it myself  github.com/deeplook/ipyrest. ;)

**Comment by martinRenou** [comment]
That's really cool :)
We should create an `awesome-ipywidgets-libraries` repository, and put the list of widgets libraries with links to repositories and binder links. That would definitely be useful for people to know what's available.

**Comment by deeplook** [comment]
Concerning your binder, this did the trick for me. Used in any notebook (then start the leaflet example in a fresh kernel):

```
! pip install ipyleaflet
! jupyter nbextension enable --py --sys-prefix ipyleaflet
```

**Comment by martinRenou** [comment]
> We should create an awesome-ipywidgets-libraries repository

There is one actually! Not specific to widgets libraries but some of them are in the list: https://github.com/markusschanta/awesome-jupyter
